README.md

```diff
@@ -20,7 +20,13 @@ All the good stuff is in **./app**. Start at **./app/main.cjsx**
 
 While editing, do your best to follow style and architecture conventions already used by the project, unless you have a reason not to. **If in doubt, ask.**
 
-## Development with Docker
+#### What to do if it doesn't run
+
+Try `rm -rf ./node_modules && npm install` to freshen up your dependencies. And read the warnings, they should should tell you if you're using the wrong version of Node or npm or if you're missing any dependencies.
+
+When running webpack, look out for type errors from [Babel Typecheck](https://github.com/codemix/babel-plugin-typecheck). You might need to add the correct [flowtype annotation](https://flowtype.org/docs/type-annotations.html) in manually.
```

Comment:
I think this is outdated now. We might even have removed typecheck.

Reply:
Yep. `babel-plugin-typecheck` isn't installed any more, so this paragraph can go.

Reply:
mind pushing the changes on this PR?